development/process.md

```diff
@@ -2,54 +2,54 @@
 layout: page
 menu_id: development
 title: Development - Process
+slack: https://join.slack.com/t/prestosql/shared_invite/enQtNTMyNzU2NzQ1NzQ4LWRhMTE4ZTA4NjM0NDA1NmFkZjEyZDJmN2MxNGY1ZTk4NmM4MzMxZDk4OGQ0NjZhNmQxMWUyNGIxMDliODk0MmU
```

Comment:
Let's move this to `_config.yaml`. I think you just put it there as `slack` and reference it in pages with `site.slack`